content/docs/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Docs Index"
+title: "Overview"
 description: "index of the docs section"
```

Comment:
All of these `description` tags need to be made more meaningful.

Reply:
I agree, but think this work can be done on a separate issue.